blocks/library/image/index.js

```diff
@@ -71,6 +71,25 @@ registerBlockType( 'core/image', {
 					return tag === 'img' || ( hasImage && ! hasText ) || ( hasImage && tag === 'figure' );
 				},
 			},
+			{
+				type: 'html',
+				isMatch( html ) {
+					const wrapper = document.createElement( 'div' );
+					wrapper.innerHTML = html;
+					return wrapper.childNodes.length === 1 && wrapper.childNodes[ 0 ].nodeName.toLowerCase() === 'img';
```

Comment:
This seems fragile. How *exact* would the HTML match need to be to create an image?